cmd/list/installed/installed.go

```diff
@@ -26,6 +26,10 @@ func List() error {
 	if err != nil {
 		return fmt.Errorf("failed to determine installation directory: %w", err)
 	}
+	latestDir, err := tool.LatestDir()
+	if err != nil {
+		return fmt.Errorf("failed to determine install directory: %w", err)
```

Comment:
```suggestion
		return fmt.Errorf("failed to determine latest/ directory: %w", err)
```
or something similar